website/docs/registries/packaging.md

```diff
@@ -0,0 +1,22 @@
+# Packaging your project
+
+When you want to share your package over package registry, it needs to be turned into an archive first. The archive will then be uploaded to the registry and downloaded by other users.
+
+Use `scarb package` command to create an archive of your package. You can read about package compression algorithm and contents in [Package tarball](/docs/registries/package-tarball) section.
+Basically when you run the command Scarb looks for the source code of your package together with metadata files such as manifest file and put them in an archive in `target/package` directory.
+
+If you are in a git repository, Scarb will first check if the repo state is clean and error out in case of any changes present in a git working directory. To ignore this check you can use the `--allow-dirty` flag.
```

Comment:
```suggestion
If you are in a Git repository, Scarb will first check if the repo state is clean and error out in case of any changes present in a git working directory. To ignore this check you can use the `--allow-dirty` flag.
```